Kin Unity/Assets/Plugins/Android/mainTemplate.gradle

```diff
@@ -26,7 +26,8 @@ apply plugin: 'com.android.application'
 
 dependencies {
     implementation fileTree(dir: 'libs', include: ['*.jar'])
-	implementation 'com.github.kinecosystem:kin-sdk-android:1.0.3.1'
+        implementation 'com.github.kinecosystem.kin-sdk-android:kin-sdk-lib:1.0.5'
+        implementation 'com.github.kinecosystem.kin-sdk-android:kin-backup-and-restore-lib:1.0.5'
```

Comment:
do we always take backup and restore in kin unity plugin? isn't it modularized like in sdk?

Reply:
No, backup and restore is always in.